### DISCUSSION TITLE
recipes/sauces-and-dressings/butternut-squash-sauce/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1190] [Comments]
# recipes/sauces-and-dressings/butternut-squash-sauce/

A collection of my recipes.

https://nicholaswilde.io/recipes/sauces-and-dressings/butternut-squash-sauce/

**Comment by nicholaswilde** [comment]
Fry Julian sage with red pepper flakes and add to sauce
